core/trino-main/src/main/java/io/trino/FeaturesConfig.java

```diff
@@ -366,6 +373,19 @@ public FeaturesConfig setExchangeCompressionCodec(CompressionCodec exchangeCompr
         return this;
     }
 
+    @Config("experimental.blockserde-vectorized-null-suppression-strategy")
+    @ConfigDescription("Strategy used for vectorized null suppression in block serde")
```

Comment:
Enable using Java Vector API for faster serialization and deserialization of exchange data